news/2023/2023-07-06-introducing-new-beatmap-pack-categories.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Introducing New Beatmap Pack Categories"
+date: 2023-07-06 09:00:00 +0000
+---
+
+Check out the latest and greatest new beatmap pack categories!
+
+![](/wiki/shared/news/2023-07-06-introducing-new-beatmap-pack-categories/banner.png)
+
+I like maps. You like maps. We all like maps.
+
+For this reason, the mad scientists over at osu! labs have been synthesizing new ways to pump high-quality beatmaps directly into your bloodstream, and their latest creations are sure to excite you!
+
+Introducing three new categories of mappack!
+
+![](/wiki/shared/news/2023-07-06-introducing-new-beatmap-pack-categories/new-beatmap-pack-categories.png)
+
+## Project Loved packs
+
+Project Loved has been bringing some of osu!'s finest creations to your screens since 2016, and the time has come to bring some *HIGH DENSITY LOVED GOODNESS* to your songs folder. Each pack contains a selection of each season's greatest hits, as chosen by the Loved Captains themselves!
+
+Show them some *love* over at the [Project Loved pack listing](https://osu.ppy.sh/beatmaps/packs?type=loved), and perhaps even earn some medals while you're at it!
+
+## Tournament packs
+
+For years people have been asking the question, *"Where can I find a nice selection of easy-to-download mappools from past tournaments?"*. Turns out the answer has been found, as Tournament Packs are now alive and kicking! Enjoy some sensible Quarter-Final pools, or test your limits with a Grand Final pool from any of the most hype tournaments to have graced the game.
+For now, the focus is on getting all of the World Cup mappools uploaded, but expect to see other tournaments joining them very soon!
+
+Give yourself a challenge over at the [tournament pack listing](https://osu.ppy.sh/beatmaps/packs?type=tournament) today!
+
+## Centurion packs
+
+To this day there are 14 mappers who have reached the incredible milestone of reaching **one hundred** ranked maps, so after some 'light interrogation', we bring you the Centurion Packs! Each pack is made up of ten of the mapper's proudest creations, nicely packaged up into a .zip for your playing pleasure!
+
+These packs can be found in the [themed pack listing](https://osu.ppy.sh/beatmaps/packs?type=theme)!
+
+---
+
+In addition to all of this, we would like to welcome [Shige-Tori[a]](https://osu.ppy.sh/users/4459449) as the new maintainer of the regular beatmap packs! (may he soon escape spreadsheet hell 🙏)
```

Comment:
This could either be reincorporated into the sentence as a joke or scrapped entirely, I don't feel like it makes any sense without context
```suggestion
In addition to all of this, we would like to welcome [Shige-Tori[a]](https://osu.ppy.sh/users/4459449) as the new maintainer of the regular beatmap packs!
```